.travis.yml

```diff
@@ -12,6 +12,7 @@ virtualenv:
   system_site_packages: true
 before_install:
   - 'for e in /.packer-env/*; do echo -n "${e}: "; cat "${e}"; done'
+  - printenv | sort # dump Travis environment for debugging
```

Comment:
Yes, the env var dump should stay. It doesn't add to build time, the output folds automatically in the web interface, and it's invaluable in case of a failure. Rather than having to add it ad-hoc in the future, I just want it in there.